examples/lazytensor_tanh.py

```diff
@@ -0,0 +1,83 @@
+"""
+Example of taking a Lazy Tensor computation and compiling it using npcomp.
+
+This example depends on the Lazy Tensor Core (LTC) of Pytorch. For information
+on how to obtain LTC, see here:
+https://github.com/pytorch/pytorch/blob/lazy_tensor_staging/lazy_tensor_core/QUICKSTART.md
+
+To run the example, make sure the following are in your PYTHONPATH:
+    1. /path/to/torch-mlir/examples
+    2. /path/to/pytorch/lazy_tensor_core
+    3. /path/to/torch-mlir/external/torch-mlir/python
+
+then, simply call `python lazytensor_tanh.py`.
+
+Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+See https://llvm.org/LICENSE.txt for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""
+
+import numpy as np
+import torch
+import lazy_tensor_core as ltc
+from torch._C import CompilationUnit
+
+from torch_mlir_e2e_test.linalg_on_tensors_backends.refbackend \
+    import RefBackendLinalgOnTensorsBackend
+from torch_mlir.passmanager import PassManager
+
+from utils.annotator import Annotation
+from utils.torch_mlir_types import TorchTensorType
+from lazytensor.builder import build_module
+
+ltc._LAZYC._ltc_init_ts_backend()
+device = 'lazy'
+dtype = torch.float32
+shape = (2, 3)
+
+x = torch.randn(shape, device=device, dtype=dtype)
+y = torch.randn(shape, device=device, dtype=dtype)
+
+# Capture lazy computation and convert to TorchScript IR
+graph_str = ltc._LAZYC._get_ltc_tensors_backend([x.tanh() * y.tanh()])
```

Comment:
what determines the order of arguments to the TorchScript function that gets created? It seems like nothing here knows whether we want `x` or `y` to be the first argument.

Reply:
I am not completely sure. From experimentation, the evaluation order does not seem to be from left to right all the time. For example, `x.tanh() * y` captures `y` as the first argument. I will look into ways of controlling the order